framework-plugins/lisk-framework-chain-connector-plugin/test/unit/inbox_update.spec.ts

```diff
@@ -86,6 +86,7 @@ describe('inboxUpdate', () => {
 			},
 			messageFeeTokenID: Buffer.from('04000001', 'hex'),
 			partnerChainOutboxRoot: Buffer.alloc(2),
+			minReturnFeePerByte: BigInt(0),
```

Comment:
this seems to be required change, but im not sure why it was not caught in the test run or in action

Reply:
Baffled me as well especially since `minReturnFeePerByte` is not an optional property in the interface.